docs/release-notes.asciidoc

```diff
@@ -24,7 +32,104 @@
 * <<rn-7-0-1>>
 * <<rn-7-0-0>>
 
+[discrete]
+[[rn-8-0-1]]
+=== 8.0.1
+
+* use of `NoNodeAvailableException` exception in endpoints, fixing
+  https://github.com/elastic/elasticsearch-php/issues/1209[#1209]
+  
+
+[discrete]
+[[rn-8-0-0]]
+=== 8.0.0
+
+This new major version of elasticsearch-php contains a brand new implementation
+compared with 7.x. It supports https://www.php-fig.org/psr/psr-7/[PSR-7] for HTTP
+messages and https://www.php-fig.org/psr/psr-18/[PSR-18] for HTTP client
+communications. We used the https://github.com/elastic/elastic-transport-php[elastic-transport-php]
+library for HTTP communications.
+
+We tried to reduce the BC breaks as much as possible with 7.x but there are some
+(big) differences:
+
+* we changed the namespace, now everything is under `Elastic\Elasticsearch`;
+* we changed the Exception model, using the namespace `Elastic\Elasticsearch\Exception`.
+  All the exceptions extends the `ElasticsearchException` interface, as in 7.x;
+* we changed the response type of each endpoints using an Elasticsearch response class.
+  This class wraps a a PSR-7 response allowing the access of the body response
+  as array or object. This means you can access the API response as in 7.x, no BC break here!
+
+You can have a look at the https://github.com/elastic/elasticsearch-php/blob/8.0/BREAKING_CHANGES.md[BREAKING_CHANGES]
+online document for more information.
+
+
+[discrete]
+[[rn-7-17-0]]
+=== 7.17.0
+
+* Allow psr/log v3
+  https://github.com/elastic/elasticsearch-php/pull/1184
+
+
+[discrete]
+[[rn-7-16-0]]
+=== 7.16.0
+
+* Added support of includePortInHostHeader in ClientBuilder::fromConfig
+  https://github.com/elastic/elasticsearch-php/pull/1181
+* Fixed UTF-16 issue in SmartSerializer with single unpaired surrogate in unicode escape
+  https://github.com/elastic/elasticsearch-php/pull/1179
+* Replace trait with abstract class to avoid Deprecated Functionality issue in PHP 8.1
+  https://github.com/elastic/elasticsearch-php/pull/1175
```

Comment:
```suggestion
* Added support of includePortInHostHeader in ClientBuilder::fromConfig
  https://github.com/elastic/elasticsearch-php/pull/1181[#1181]
* Fixed UTF-16 issue in SmartSerializer with single unpaired surrogate in unicode escape
  https://github.com/elastic/elasticsearch-php/pull/1179[#1179]
* Replace trait with abstract class to avoid Deprecated Functionality issue in PHP 8.1
  https://github.com/elastic/elasticsearch-php/pull/1175[#1175]
```